fastestimator/backend/_focal_loss.py

```diff
@@ -47,13 +93,14 @@ def focal_loss(y_true: Tensor, y_pred: Tensor, gamma: float = 2.0, alpha: float
     ```
 
     Args:
-        y_true: Ground truth class labels which should take values of 1 or -1.
+        y_true: Ground truth class labels which should take values of 1 or 0.
         y_pred: Prediction score for each class, with a shape like y_true. dtype: float32 or float16.
         alpha: Weighting factor in range (0,1) to balance
                 positive vs negative examples or (-1/None) to ignore. Default = 0.25
         gamma: Exponent of the modulating factor (1 - p_t) to
                balance easy vs hard examples.
-        from_logits: Whether y_pred is logits (without softmax).
+        normalize: Whether to normalize focal loss along samples based on number of positive anchors per samples.
```

Comment:
let's not use the word `anchors`, as it is only specific to object detection.  Focal loss is more general than that